WPFHospitalEditor/AdditionalInformation.xaml.cs

```diff
@@ -29,15 +30,15 @@ public partial class AdditionalInformation : Window
         private IEnumerable<MedicationDto> allMedications;
         private DynamicGridControl gridControl;
 
-        public AdditionalInformation(MapObject mapObject, Building building, Role role)
+        public AdditionalInformation(MapObject mapObject, Building building)
         {
             InitializeComponent();
             this.mapObject = mapObject;
             this.building = building;
             this.descriptionParts = mapObject.Description.Split("&");
             this.contentRows = descriptionParts[1].Split(";");
             this.oldMapObject = mapObject;
-            this.role = role;
+            this.role = HospitalMap.role;
             this.allEquipment = equipmentServerController.GetEquipmentByRoomId(mapObject.Id);
```

Comment:
AdditionalInformation and Building classes has field role. That need to be deleted because HospitalMap.role is static field. Use static field wherever is needed.